packages/core/src/repositories/documentVersionsRepository/index.ts

```diff
@@ -233,16 +233,14 @@ export class DocumentVersionsRepository extends Repository<
             mergedAt: max(this.scope.mergedAt).as('maxMergedAt'),
           })
           .from(this.scope)
-          .where(and(filterByMaxMergedAt(), filterByproject()))
+          .where(and(filterByMaxMergedAt(), filterByProject()))
           .groupBy(this.scope.documentUuid),
       )
 
     const documentsFromMergedCommits = await this.db
       .with(lastVersionOfEachDocument)
       .select(this.scope._.selectedFields)
       .from(this.scope)
-      .innerJoin(documentVersions, eq(this.scope.id, documentVersions.id))
-      .innerJoin(projects, eq(this.scope.projectId, projects.id))
```

Comment:
i think these two aren't needed, tests pass correctly and cannot see anything wrong when testing locally, let me know if I'm missing something @andresgutgon @csansoon